packages/client/src/views/SysAdmin/Team/user/UserList.tsx

```diff
@@ -466,25 +466,39 @@ function UserListComponent(props: IProps) {
   }
 
   function getStatusMenuType(
+    userDetails: IUserDetails | null,
+    locationId: string,
     user: GQLUser,
```

Comment:
Could you get the user id from `userDetails`? Could this parameter be dropped altogether?

Reply:
I thought userDetails is for logged in user's details, and user is the information for the users in the userlist rows. they are not same. so I don't think this parameter can be dropped.